servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/OutlierDetectorConfig.java

```diff
@@ -304,8 +303,8 @@ public Duration maxEjectionTime() {
      * are ejected at the time.
      * @return the maximum amount of jitter to add to the ejection time.
      */
-    public Duration maxEjectionTimeJitter() {
-        return maxEjectionTimeJitter;
+    public Duration ejectionTimeJitter() {
```

Comment:
Do we need to update javadoc that says "The maximum amount of jitter" and remove "maximum" from there?